src/components/postView/view/postDisplayView.js

```diff
@@ -20,8 +22,10 @@ import { ParentPost } from '../../parentPost';
 // Styles
 import styles from './postDisplayStyles';
 import { OptionsModal } from '../../atoms';
+import { orientations } from '../../../redux/constants/orientationsConstants';
 
 const HEIGHT = Dimensions.get('window').width;
```

Comment:
@aliseyalvi height is equal to width here, could be a typo

Reply:
it coule de a typo